### PR TITLE
Geo map: Reset opacity on layer unselect

### LINKED ISSUE #1889: Geo up: when unselect layer reset UI state opacity to default (1)

### PR DESCRIPTION
**Pull request by joonasnuutinen**
Resolves #1889 

https://user-images.githubusercontent.com/18427577/207977717-277385ca-fb79-4540-afe1-0077987a7b18.mov

Draft because merging #1902 will probably cause conflicts.

### CODE REVIEW

**Review by eraviolo — APPROVED** [review]

File: src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/MapVisualizerPanel.tsx
Comment (minotogna) [inline src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/MapVisualizerPanel.tsx]: Dispatching double action in this case could be avoided. you can simply set the opacity to 1 in removeForestLayer reducer
state.forestOptions.opacity[key] = 1

Comment (joonasnuutinen) [reply src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/MapVisualizerPanel.tsx]: @minotogna Good catch! Fixed now.

Comment (minotogna) [reply src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/MapVisualizerPanel.tsx]: Thanks @joonasnuutinen . Is this pr good to be merged or is there a reason why it’s still in draft ?

Comment (joonasnuutinen) [reply src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/MapVisualizerPanel.tsx]: @minotogna If #1902 had been merged before this one, there would've been merge conflicts. However, we can merge this first and fix the conflicts in #1902 instead.

Comment (minotogna) [reply src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/MapVisualizerPanel.tsx]: ah ok, waiting for @eraviolo review on the other one .

**Review by minotogna — APPROVED** [review]

**Comment by sorja** [comment]
Merge conflicts @joonasnuutinen

**Pull request merged**
